### PR TITLE
Notification API Change

### PR DESCRIPTION
**Pull request by dishanest**
## Overview
Changed notification handling according to new APNs payload schema. 

## Test Coverage
Playtest on physical device

### CODE REVIEW

File: Volume/Views/Onboarding/FollowView.swift [hunk 1]
```diff
@@ -24,15 +24,13 @@ extension OnboardingView {
                         print(error.localizedDescription)
                     }
                 }, receiveValue: { value in
-                    state = .results([Publication](value))
+                    let publications = [Publication](value)
+                    state = .results(publications)
                 })
         }
-
+
```
Comment (danielVebman) [inline Volume/Views/Onboarding/FollowView.swift:31]: delet whitespace

File: Volume/Views/Onboarding/FollowView.swift [hunk 2]
```diff
@@ -24,15 +24,13 @@ extension OnboardingView {
                         print(error.localizedDescription)
                     }
                 }, receiveValue: { value in
-                    state = .results([Publication](value))
+                    let publications = [Publication](value)
+                    state = .results(publications)
```
Comment (danielVebman) [inline Volume/Views/Onboarding/FollowView.swift:28]:
```suggestion
                    state = .results([Publication](value))
```

File: Volume/Views/Onboarding/FollowView.swift [hunk 3]
```diff
@@ -45,24 +43,29 @@ extension OnboardingView {
...
+                .background(
+                    GeometryReader { proxy in
+                        let offset = -proxy.frame(in: .named(scrollViewCoordinateSpace)).origin.y
+                        Color.volume.backgroundGray
+                            .preference(key: OffsetPreferenceKey.self, value: offset)
+                    }
+                )
+            }
+            .coordinateSpace(name: scrollViewCoordinateSpace)
+            .onPreferenceChange(OffsetPreferenceKey.self) {
```
Comment (danielVebman) [inline Volume/Views/Onboarding/FollowView.swift:56]: Could you explain what this is doing? Also, are you using TrackableScrollView anymore anywhere in the app? If not then can we delete the file?

**Review by danielVebman — APPROVED** [review]

File: Volume/Views/Onboarding/FollowView.swift (hunk 3 above)
Comment (dishanest) [reply Volume/Views/Onboarding/FollowView.swift:56]: I actually made a GitHub mistake on this PR by starting this branch from the one I submitted in the onboarding scroll fix PR, so I'll address this issue in the that PR page.

**Pull request merged**